schema_utils_test.py

```diff
@@ -886,33 +888,32 @@ def test_dict_schema(self):
         self.check_normalization(
             schema, mappings, invalid_values_with_error_messages)
 
-    def test_notification_email_list_validator(self):
```

Comment:
Why remove the email list validator? Is that not needed anymore? ``is_valid_email`` is still in schema_utils from what I can tell.

Reply:
Ï forgot to remove the `is_valid_email`, it is removed now.